src/lib/dom.iterable.d.ts

```diff
@@ -5,9 +5,17 @@ interface DOMTokenList {
 }
 
 interface NodeList {
+    keys(): IterableIterator<number>;
+    values(): IterableIterator<[number, Node]>;
+    entries(): IterableIterator<Node>;
+    forEach(): void;

```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach, this should be:

```ts
    /**
      * Performs the specified action for each node in an list.
      * @param callbackfn  A function that accepts up to three arguments. forEach calls the callbackfn function one time for each element in the list.
      * @param thisArg  An object to which the this keyword can refer in the callbackfn function. If thisArg is omitted, undefined is used as the this value.
      */
     forEach(callbackfn: (value: Node, index: number, listObj: NodeList) => void, thisArg?: any): void;
```